addons/mail/models/discuss/discuss_channel.py

```diff
@@ -467,7 +467,7 @@ def _sync_field_names(self):
         res = super()._sync_field_names()
         res[None] += [
             Store.Attr("avatar_cache_key", predicate=is_channel_or_group),
-            Store.One("discuss_category_id", ["name"], sudo=True),  # sudo: discuss.category - reading name is acceptable
+            Store.One("discuss_category_id", ["name", "sequence"], sudo=True),  # sudo: discuss.category - reading name and sequence is acceptable
```

Comment:
I would move it to a separate line, and adapt it in a way where we don't need to update the comment every time we change the fields.
Something like "guests can read categories of accessible channels"